### PR TITLE
Update wechatwebdevtools from 1.05.2203070 to 1.05.2204250

### PR DESCRIPTION
**Pull request by baocang**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/wechatwebdevtools.rb [hunk 1]
```diff
@@ -18,8 +18,8 @@
   app "wechatwebdevtools.app"
 
   zap trash: [
-    "~/Library/Application Support/微信web开发者工具",
-    "~/Library/Caches/微信web开发者工具",
```
Comment (baocang) [inline Casks/wechatwebdevtools.rb:22]: These directories do not exist, causing --zap to fail to delete files

File: Casks/wechatwebdevtools.rb [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-  version "1.05.2203070"
-  sha256 "8704f9b8f07bf823268906ee004c5c77e34773aa86b17b3260cec9f3dc663a30"
+  version "1.05.2204250"
+  sha256 "d46828cfa36bc0a084503f671e6f3bdfde6031f4b73e90c5dc9e8dae9d62c559"
 
   url "https://dldir1.qq.com/WechatWebDev/release/p-ae42ee2cde4d42ee80ac60b35f183a99/wechat_devtools_#{version}.dmg"
-  name "wechat web devtools"
-  name "微信web开发者工具"
-  desc "Weixin DevTools for Official Account and Mini Program development"
-  homepage "https://mp.weixin.qq.com/debug/wxadoc/dev/devtools/download.html"
```
Comment (baocang) [inline Casks/wechatwebdevtools.rb:9]: This page has been redirected to https://developers.weixin.qq.com/miniprogram/dev/devtools/download.html

File: Casks/wechatwebdevtools.rb [hunk 3]
```diff
@@ -18,8 +18,8 @@
   app "wechatwebdevtools.app"
 
   zap trash: [
-    "~/Library/Application Support/微信web开发者工具",
-    "~/Library/Caches/微信web开发者工具",
+    "~/Library/Application Support/微信开发者工具",
+    "~/Library/Caches/微信开发者工具",
```
Comment (baocang) [inline Casks/wechatwebdevtools.rb:22]: These directories do not exist, causing --zap to fail to delete files

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @baocang.

**Pull request merged**